sql-database/failover-groups/add-single-db-to-failover-group-az-cli.sh

```diff
@@ -0,0 +1,114 @@
+﻿#!/bin/bash
+# Set variables
+subscriptionID=<SubscriptionID>
```

Comment:
This value is no longer used by the script and should be removed. Or you can add the `az account set` command back in. The only suggested change was to avoid `az login` due to the fact that it's interactive. `az account set` is not.